deploy/crds/noobaa.io_v1alpha1_backingstore_cr.yaml

```diff
@@ -2,4 +2,6 @@ apiVersion: noobaa.io/v1alpha1
 kind: BackingStore
 metadata:
   name: default
+  labels:
+    backingstore: noobaa
```

Comment:
I think we should avoid setting a default label for the BackingStore. Instead, users should have the flexibility to add the label as needed.